.changeset/real-needles-study.md

```diff
@@ -0,0 +1,7 @@
+---
+'@talend/bootstrap-sass': patch
```

Comment:
Was wondering if we were supposed to change this package
Seems like it's a fork for technical reasons but if anything needs to change , it should be an override in @talend/bootstrap-theme package

Reply:
It was generated by the command `yarn changeset`. Maybe you are right and this update is not needed :thinking:

Reply:
Gonna revert part of changes